frontend/src/components/UserDetails.vue

```diff
@@ -129,15 +137,19 @@ export default {
           org_details: null,
           access_team_notes: null
         },
+        allRoles: null,
         federatedIdentities: null
       }
     };
   },
   async created() {
+    //Create global ref to allow role update from UserUpdateRoles
+    this.$root.$refs.UserDetails = this;
```

Comment:
I don't really know how to do better than this. My guess is something like putting the data into Vuex, but I'm really familiar with it. Maybe @tschia has a suggestion if it's worth the time.